### DISCUSSION TITLE
Photos in Hidden folder, Thumbnail location 'not a directory'

### DISCUSSION THREAD

**Discussion by locqust** [photoprism/photoprism#2069] [Q&A] [answered]
Not sure if this has been discussed before, couldn't find anything being mentioned about it.

I'm running PhotoPrism 220121-2b4c8e1f-Linux-x86_64 on Docker for Desktop, 4.4.3. Originals folders on a Windows drive shared into the container. Over 150K photos. 
I have about 200 photos in my hidden folder, some have Thumbnails, others have the broken icon. But they all have the same error similar to this...

Open /Photoprism/Storage/Cache/Thumbnails/D/1/6/D16668c010779d140a1634aa6d83f6365a5df9c1_2048x2048_fit.Jpg: Not A Directory

Yet if I browse to that location in Windows the folder exists and has plenty of other thumbnails in there. (folders are named lowercase)
The originals that have this error are a mix of photos taken on a modern smartphone, in the last few days/months. Photos taken on an old digital camera about 10 years ago and scans of old 24 shot camera films. 
Original index was performed last September when I first installed PhotoPrism. WebDAV is being used to import new photos as they are taken.
Not quite sure what I can do to fix it?

**Comment by lastzero** [comment, marked as answer]
https://docs.photoprism.app/getting-started/troubleshooting/#broken-thumbnails
